Jenkinsfile_parameterized

```diff
@@ -15,10 +15,14 @@ properties([
   pipelineTriggers([[$class: 'GitHubPushTrigger']])
 ])
 
-List<LinkedHashMap<String, Object>> secrets = [
-  secret('s2s-secret-for-tests', 'S2S_SECRET'),
-  secret('idam-users-bulkscan-password', 'IDAM_USER_PASSWORD'),
-  secret('idam-client-secret', 'IDAM_CLIENT_SECRET'),
+def secrets = [
+  's2s-${env}': [
+    secret('microservicekey-secret-for-tests', 'S2S_SECRET')
```

Comment:
is this the correct key ?

Reply:
oh nopes! pattern was correct just minute ago on another repo. too quick to copy/paste here 😉